openshift/ci-operator/source-image/Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM src
 
 COPY oc /usr/bin/oc
-COPY serving ${GOPATH}/src/knative.dev/serving
-COPY eventing ${GOPATH}/src/knative.dev/eventing
+
+COPY --from=registry.svc.ci.openshift.org/openshift/knative-v0.13.3:knative-serving-src ${GOPATH}/src/knative.dev/serving/ ${GOPATH}/src/knative.dev/serving/
```

Comment:
should we wait for updates landed w/ the 0.14x bumps for eventing/serving ?

Reply:
Actually. You could put this directly in your PR for updating eventing or serving and use 0.14 there